src/Upgrade/Infrastructure/Adapter/IntegratorAdapter.php

```diff
@@ -11,6 +11,7 @@
 
 use Core\Infrastructure\Service\ProcessRunnerServiceInterface;
 use Upgrade\Application\Adapter\IntegratorAdapterInterface;
+use Upgrade\Application\Dto\IntegratorResponseDto;
 use Upgrade\Application\Dto\StepsResponseDto;
 
 class IntegratorAdapter implements IntegratorAdapterInterface
```

Comment:
btw,  it not an adapter :) seems like we already use that

Reply:
renamed to IntegratorExecutor